weltyt.py

```diff
@@ -44,6 +44,33 @@ def buy_plants(plants):
 
         print("Invalid input, write integer only please. ")
 
+def buy_pesticides(pesticides):
+    amount = input("How many pesticides do you want to buy? Write only integer please. ")
+
+    try:
+        amount = int(amount)
+
+        money_amount = cur.execute(
```

Comment:
DRY https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
We used exactly the same code before - move it to some method or at least assign the SQL to some kind of constant.